hack/install_operator.yaml

```diff
@@ -1,19 +1,9 @@
 # This file was autogenerated by github.com/codeready-toolchain/api/olm-catalog.sh'
----
```

Comment:
why is this CR removed?

Reply:
We remove OperatorGroup because the toolchain-operator is going to be installed only in AllNamespaces mode => in the default namespace "openshift-operators" that the OperatorGroup contains by default. So there is no need to create another one

Reply:
ok, thanks for the clarification